### PR TITLE
Create new-doc.md

### PR DESCRIPTION
**Pull request by femkreations**
Initial version of the new doc template

### CODE REVIEW

**Review by zzap — COMMENTED** [review]
Are we going to have labels for these issues?

File: .github/ISSUE_TEMPLATE/new-doc.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: New Doc Request
+about: Propose a new page in the WordPress documentation.
+
```
Comment (zzap) [inline .github/ISSUE_TEMPLATE/new-doc.md:4]: We have `new document` label now so you can add it here. Thanks

**Review by zzap — CHANGES_REQUESTED** [review]

**Comment by theMikeD** [comment]
`Please describe the new page that you are requesting.` seems a little more straightforward.

**Review by femkreations — COMMENTED** [review]
Updating the description in the first field

**Review by zzap — APPROVED** [review]

**Pull request merged**